### PR TITLE
Sort strings in translation bundle in alphabet

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

File: g11n-ws/modules/md-data-api-bundleimpl/src/main/java/com/vmware/vip/messages/data/dao/impl/OneComponentDao.java [hunk 1]
```diff
@@ -23,130 +23,129 @@
...
-	
-	private static Logger logger = LoggerFactory.getLogger(OneComponentDao.class);
-	
-	@Autowired
-	private BundleConfig bundleConfig;
-	
-	
-	
-	@Override
-	public ResultI18Message get(String productName, String version, String component, String locale) throws DataException{
```
Comment (linr211) [inline g11n-ws/modules/md-data-api-bundleimpl/src/main/java/com/vmware/vip/messages/data/dao/impl/OneComponentDao.java]: why need to modify this method?

Comment (tigershi) [reply g11n-ws/modules/md-data-api-bundleimpl/src/main/java/com/vmware/vip/messages/data/dao/impl/OneComponentDao.java]: I have checked it, it need to sort the bundle file content when sync the translations from GRM

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/dao/impl/SourceDaoImpl.java [hunk 2]
```diff
@@ -39,172 +38,158 @@
...
-				+ ConstantsChar.BACKSLASH
-				+ ResourceFilePathGetter
-						.getLocalizedJSONFileName(ConstantsKeys.LATEST);
-		LOGGER.info("Read content from file: {}{}",  basepath, filepath);
+   private static Logger LOGGER = LoggerFactory.getLogger(SourceDaoImpl.class);
+   @Autowired
+   private SqlLiteDao sqlLite;
+
+   @Override
+   public String getFromBundle(SingleComponentDTO singleComponentDTO, String basepath) {
```
Comment (linr211) [inline g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/dao/impl/SourceDaoImpl.java]: I don't understand this change. Will ping you offline.

Comment (tigershi) [reply g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/dao/impl/SourceDaoImpl.java]: this just formats it and  just change the write JSON file

**Pull request merged**